src/main/java/org/terasology/inGameHelp/components/HelpItem.java

```diff
@@ -20,13 +20,31 @@
 import org.terasology.rendering.nui.widgets.browser.data.basic.HTMLLikeParser;
 import org.terasology.rendering.nui.widgets.browser.data.html.HTMLDocument;
 
+/**
+ * Interface for adding help information to a document
+ */
 public interface HelpItem {
```

Comment:
this interface defines what a help item is made of, i.e., that it has a title, a category, paragraphs, ...